sales.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <link rel='stylesheet' type='text/css' href='style.css'>
+    <title>Sales</title>
+  </head>
+  <body>
+    <header id='header'>
+      <h1>Projected Sales</h1>
+      <nav>
+        <ul>
+          <li><a href='index.html'>Home</a></li>
+          <li><a href='sales.html'>Sales</a></li>
+          <li><a href='styleguide.html'>Style</a></li>
+        </ul>
+      </nav>
+    </header>
+    <table id='salesTable'></table>
+    <form id='addStore'>
+      <fieldset>
+        <legend><h3>Add a store</h3></legend>
+        <label> Store Name:
+          <input type='text' id='name'>
+        </label> <br>
```

Comment:
I would recommend not using `<br>` tags, and rather using a `display: inline-block` or other CSS property to manage the layout of your HTML.